Sources/PrimerSDK/Classes/Extensions & Utilities/IntExtension.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  IntExtension.swift
+//  PrimerSDK
+//
+//  Created by Carl Eriksson on 14/09/2021.
+//
+
+internal extension Int {
+    func toCurrencyString(currency: Currency) -> String {
+        switch currency {
+        case .USD: return String(format: "$%.2f", Float(self) / 100)
```

Comment:
This (i.e. `String(format: "$%.2f", Float(self) / 100)`) can lead to wrong amount, due to floating point.

Can you check how to do that using a `NumberFormatter`?

```
let formatter = NumberFormatter()
formatter.numberStyle = .currency
```

Also an idea, we could add the currency symbols in the `Currency` structure we already have, and use them from there.